libs/community/langchain_community/document_loaders/oracleadb_loader.py

```diff
@@ -0,0 +1,123 @@
+from typing import Any, Dict, List, Optional
+
+from langchain_core.documents import Document
+
+from langchain_community.document_loaders.base import BaseLoader
+
+
+class AutonomousDatabaseLoader(BaseLoader):
+    """
+    Load from oracle adb
+
+    Autonomous Database connection can be made by either connection_string
+    or tns name. wallet_location and wallet_password are required
+    for TLS connection.
+    Each document will represent one row of the query result.
+    Columns are written into the `page_content` and 'metadata' in
+    constructor is written into 'metadata' of document,
+    by default, the 'metadata' is None.
+    """
+
+    def __init__(
+        self,
+        query: str,
+        user: str,
+        password: str,
```

Comment:
nit could we add `*,` between password and schema, to enforce that schema onwards should be specified as named args

Reply:
Sure, I will add *.